Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -552,19 +551,38 @@ async function redirectWhenTicketIsInvalid(
 
     try {
         if (ticketIsValid(ticket)) {
-            logger.debug(`Router -check existing ticket`);
+            logger.debug(`Router - check existing Processed ticket`);
```

Comment:
Nit: debug statement should say it's checking an "existing Processed ticket that's not expired".

Expired processed tickets are handled in the else block.